tests/performance-test/performance-test.sh

```diff
@@ -266,25 +266,25 @@ while true; do
         ;;
         "RESULTS")
             printf "\n*** Collecting test results ***\n"
-            # PODNAME=$(oc get pod -l job-name=stf-perftest-notify -o jsonpath='{.items[0].metadata.name}')
-            # oc exec "$PODNAME" -- pkill collectd > /dev/null
-            # sleep 3
-            # oc cp "${PODNAME}:/tmp/events.json" /tmp/events.json
+            PODNAME=$(oc get pod -l job-name=stf-perftest-notify -o jsonpath='{.items[0].metadata.name}')
+            oc exec "$PODNAME" -- pkill collectd > /dev/null
+            sleep 3
+            oc cp "${PODNAME}:/tmp/events.json" /tmp/events.json
 
-            # # get total number of events generated by collectd
-            # NUM_COLLECTD_EVENTS=$(wc -l /tmp/events.json | awk '{ print $1 }')
+            # get total number of events generated by collectd
+            NUM_COLLECTD_EVENTS=$(wc -l /tmp/events.json | awk '{ print $1 }')
 
-            # # get number of events seen by elasticsearch
-            # get_es_event_count
+            # get number of events seen by elasticsearch
+            get_es_event_count
 
-            # # calulate success ratio
-            # SUCCESS_RATIO=$(( ES_EVENT_RECV_COUNT / NUM_COLLECTD_EVENTS)).$(( (ES_EVENT_RECV_COUNT * 100 / NUM_COLLECTD_EVENTS) % 100 ))
+            # calulate success ratio
+            SUCCESS_RATIO=$(( ES_EVENT_RECV_COUNT / NUM_COLLECTD_EVENTS)).$(( (ES_EVENT_RECV_COUNT * 100 / NUM_COLLECTD_EVENTS) % 100 ))
```

Comment:
huh, I learned something new today about bash syntax!

Reply:
Even after all these years it seems like every day is discover-a-feature-of-bash day.